Char to bit

hevennnnnn opened the pull request:
I did a simple array structure wrapped in a static object that has an index from 0 to 63. (6 bit integer range  💥😳)

removed what I wrote in the readme except the intro

hevennnnnn committed:
miaw

hevennnnnn committed:
finished the mapping class

hevennnnnn committed:
Update README.md

hevennnnnn committed:
Update README.md

hacknstack reviewed (commented):
Where are the unit tests??? How do I know this works??? This is utterly unacceptable. (LGTM 👍)

Zep3k reviewed (commented):
Why have i been requested to review this? Also gpt coded this?

hevennnnnn commented:
> Why have i been requested to review this? Also gpt coded this?

because that's how github works. Yes as Im learning JS with chatgpt.

hevennnnnn commented:
> Where are the unit tests??? How do I know this works??? This is utterly unacceptable. (LGTM 👍)

How do I test this ??????? Ty for reviewing.  😎

Please no trolling, only serious review for improving. ty.

hevennnnnn committed:
Update BitToChar.js

hevennnnnn committed:
Added tests

hevennnnnn committed:
fixed merging issues

hevennnnnn committed:
Added Jest and tests

hevennnnnn commented:
I Just added Jest, a module to write JavaScript tests, and a simple test for the class BitToChar. I've updated the readme in there to install Node, please check it out, and try to run the test on your computer and tell me if there are issues

Please focus on classes 

BitToChar
BitToChar.test

and try running the test on your machine

hacknstack reviewed (commented):
LGTM

hevennnnnn committed:
Merge branch 'main' into CharToBite
